### ISSUE TITLE
MobileNetV3 transfer learning Hard_swish not supported

### ISSUE THREAD

**Issue by KapitanKabzon** [google-coral/edgetpu#353]
What needs to be changed in the standard MobileNetV3small implementation to be able to transfer-learn it and then run it on the USB accelerator?

Is there a TF2 model(*.h5) that already has changed the HARD_SWISH to something the Coral TPU supports?

At the moment the model takes 200ms to infer on the Accelerator, while the Coral's own Mobilenet V3(https://coral.ai/models/) takes 12ms on classify_image.py .

By using the standard tutorial https://colab.research.google.com/github/google-coral/tutorials/blob/master/retrain_classification_ptq_tf2.ipynb , i instead used the MobilenetV3small .

**Comment by Naveen-Dodda** [comment]
Hello, 

You can turn on minimalistic=True

A simple model would look like below 

base_model = tf.keras.applications.MobileNetV3Large(input_shape=IMG_SHAPE,
                                              include_top=False,
                                              minimalistic=True, 
                                              weights='imagenet')

base_model.trainable = True

model = tf.keras.Sequential([
  base_model,
  tf.keras.layers.Conv2D(filters=32, kernel_size=3, activation='relu'),
  tf.keras.layers.GlobalAveragePooling2D(),
  tf.keras.layers.Dense(units=NUM_OUTPUTS, activation='softmax')
])

optimizer = tf.keras.optimizers.Adam()

I have verified the latency, this should have latency =~3ms

**Comment by KapitanKabzon** [comment]
I have checked myself, everything works well.
Thank you very much.
